Client/sdk/game/CVehicle.h

```diff
@@ -212,6 +212,7 @@ class CVehicle : public virtual CPhysical
     virtual bool           GetTyresDontBurst() = 0;
     virtual unsigned short GetAdjustablePropertyValue() = 0;
     virtual float          GetHeliRotorSpeed() = 0;
+    virtual float          GetPlaneRotorSpeed() = 0;
```

Comment:
It's the most marvellous place to put a const modifier

Reply:
most marvellous place?
So should I put or not?

Reply:
@tederis most marvellous place?
So should I put or not?

Reply:
yes, most the time your getters should be const